tests/testthat/test-5prevalence.R

```diff
@@ -314,71 +317,71 @@ test_that("subsetByRare", {
                  "'prevalence' must be a single numeric value or coercible to one")
     # Expect TSE object
     expect_equal(class(subsetByRare(GlobalPatterns)), class(GlobalPatterns))
-    
+
     # Results compatible with getRare
     pr1 <- rownames(subsetByRare(
         GlobalPatterns, rank = "Phylum", detection=0.1/100,
         as_relative=TRUE, sort=TRUE))
-    pr2 <- getRare(GlobalPatterns, rank = "Phylum", detection=0.1/100, 
+    pr2 <- getRare(GlobalPatterns, rank = "Phylum", detection=0.1/100,
                        as_relative=TRUE, sort=TRUE)
     expect_true(all(pr1 == pr2))
-    
+
     # Retrieved taxa are the same for counts and relative abundances
     pr1 <- assay(subsetByRare(GlobalPatterns, prevalence=0.1/100, as_relative=TRUE), "counts")
     pr2 <- assay(subsetByRare(GlobalPatterns, prevalence=0.1/100, as_relative=FALSE), "counts")
     expect_true(all(pr1 == pr2))
-    
+
     # Prevalence and detection threshold at 0 has the same impact on counts and relative abundances
     pr1 <- rownames(subsetByRare(GlobalPatterns, detection=0, prevalence=0, as_relative=TRUE))
     pr2 <- rownames(subsetByRare(GlobalPatterns, detection=0, prevalence=0, as_relative=FALSE))
     expect_true(all(pr1 == pr2))
-    
+
     # subsetByRare + subsetByPrevalent should include all the taxa in OTU level
     d <- runif(1, 0.0001, 0.1)
     p <- runif(1, 0.0001, 0.5)
-    rare <- rownames(subsetByRare(GlobalPatterns, detection=d, prevalence=p, 
+    rare <- rownames(subsetByRare(GlobalPatterns, detection=d, prevalence=p,
                                       as_relative=TRUE))
-    
-    prevalent <- rownames(subsetByPrevalent(GlobalPatterns, detection=d, prevalence=p, 
+
+    prevalent <- rownames(subsetByPrevalent(GlobalPatterns, detection=d, prevalence=p,
                                                 as_relative=TRUE))
-    
+
     all_taxa <- c(rare, prevalent)
-    
```

Comment:
I'm not sure if these lines should be modified like this.

Reply:
I only modified some deprecated function names and saved the file which might have removed white spaces.

Reply:
One should always check what has been changed in the files. It makes the reviewing process easier and helps to maintain high-quality of code

Reply:
ACK.